Source/DafnyLanguageServer.Test/Various/StressVerification.cs

```diff
@@ -0,0 +1,71 @@
+﻿using Microsoft.Dafny.LanguageServer.IntegrationTest.Extensions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OmniSharp.Extensions.LanguageServer.Protocol.Client;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using OmniSharp.Extensions.LanguageServer.Protocol.Document;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+
+namespace Microsoft.Dafny.LanguageServer.IntegrationTest.Various {
+  [TestClass]
+  public class StressVerificationTest : DafnyLanguageServerTestBase {
```

Comment:
This stress test is still failing (looping forever). I am considering removing it from the test suite for now or deactivate it until we can properly fix it. I already filed an issue on it (#1913)